sandbox/pre-registration-mz.properties

```diff
@@ -19,49 +19,31 @@ logging.level.com.zaxxer.hikari=DEBUG
 # ------------ Project Version for the API response ------------ 
 version=1.0
 
-# Not in use, to be removed after verification -----------------
-application.id=pre-registration
-preregistration.preid.regex=[0-9]+
-
-
 # ------------ Age to be an adult ------------------------------
 # Used in UI to identify the applicant type 
+# use the global property for the age
 mosip.adult.age=5
```

Comment:
This should come from the main application.properties file and has to be shared across all applications